### DISCUSSION TITLE
Html component style prop type error

### DISCUSSION THREAD

**Discussion by baronkoko** [nuxt/nuxt#17277] [Questions] [answered]
Can we use the `style` property on the `<Html />` component?

The problem:
![Screenshot at Jul 15 12-25-56](https://user-images.githubusercontent.com/27996386/179195576-4c27a38a-4901-4bca-8621-20e552e7f629.png)

Reproduction: https://stackblitz.com/edit/github-jnuawp?file=app.vue

**Comment by pi0** [comment, marked as answer]
Hi. Try `:style="'foo'"` (or `style="foo"`)

**Reply by baronkoko** [reply]
So obvious 🤦‍♂️, thank you, sorry for bothering you.
Initially, it happened when I used the `<Html />` component as a root element (example https://stackblitz.com/edit/github-5tb287?file=app.vue,components%2FHtmlAdjuster.vue). We can't use it as a root element, right?

**Reply by ShaggyTech** [reply]
You can use the HTML component as root element in app.vue.

**Reply by baronkoko** [reply]
> You can use the HTML component as root element in app.vue.

Got it, many thanks
